tests/unit/lib/kwlib_test.sh

```diff
@@ -230,10 +230,10 @@ function test_cmd_manager_check_test_mode_option()
   local ret
 
   ret=$(cmd_manager 'TEST_MODE' 'pwd')
-  assertEquals "Expected pwd, but we got $ret" "$ret" "pwd"
+  assertEquals "Expected pwd, but we got $ret" "pwd" "$ret"
 
   ret=$(cmd_manager 'TEST_MODE' 'ls -lah')
-  assertEquals "Expected ls -lah, but we got $ret" "$ret" "ls -lah"
+  assertEquals "Expected ls -lah, but we got $ret" "ls -lah" "$ret"
```

Comment:
Take the opportunity to surround `ret` inside `Expected ...` with curly braces (`{}`), and replace the double quotes around the expected for single quotes